src/pages/household/settings/dependent/[uuid].svelte

```diff
@@ -1,24 +1,32 @@
 <script>
-import { dependents } from '../../../../data/dependents'
+import {
+  deleteDependent,
+  dependents,
+  initialized,
+  loadDependents,
+  updateDependent
+} from '../../../../data/dependents'
 import DependentForm from '../../../../components/DependentForm.svelte'
 import { goto } from '@roxi/routify'
 import { Page } from '@silintl/ui-components'
+import user from "../../../../authn/user";
```

Comment:
```suggestion
import user from "../../../../authn/user"
```
Remove the semicolon? Also, are imports sorted in this project?

Reply:
Yes, and yes. I'll update it, thanks.

Reply:
I probably pasted in code that used `$user`, so PhpStorm automatically added the import